gobblin-salesforce/src/main/java/org/apache/gobblin/salesforce/SalesforceConnector.java

```diff
@@ -94,4 +123,12 @@ private String getServiceBaseUrl() {
   public String getFullUri(String resourcePath) {
     return StringUtils.removeEnd(getServiceBaseUrl(), "/") + StringUtils.removeEnd(resourcePath, "/");
   }
+
+  public String getAccessToken() {
```

Comment:
The following two can be package-level access.

Reply:
fixed